lms/static/sass/multicourse/_courses.scss

```diff
@@ -426,29 +439,34 @@ $facet-background-color: #007db8;
     .header-search-facets, .header-facet {
       @extend %t-title6;
       @extend %t-strong;
-      padding: ($baseline/2);
       margin: 0;
+      padding: ($baseline/2);
       color: $facet-text-color;
       text-transform: none;
     }
 
     .header-facet {
-      margin: 0 ($baseline/2);
-      padding: ($baseline/2) 0;
-    }
-
-    .search-facets-lists section {
       border-top: 1px solid $gray-l4;
     }
 
     .facet-list {
       @extend %ui-no-list;
       @include clearfix();
-      padding-bottom: ($baseline/2); 
+      padding-bottom: ($baseline/2);
 
       &.collapse {
-        max-height: ($baseline*14);
+        max-height: ($baseline*1.5*9) - ($baseline/2);
+        overflow: hidden;
+      }
+
+      li {
```

Comment:
NIT: it would be great to use .facet-option or something similar for this instead of the element.